service/controller/resource/prometheusremotewrite/resource_test.go

```diff
@@ -233,7 +233,7 @@ func remoteWrite(name, namespace, clusterSelector string) pmov1alpha1.RemoteWrit
 	}
 }
 
-func prometheus() promv1.Prometheus {
```

Comment:
Having a look at this function, is it normal that we have a hardcoded version?

Reply:
This function is hardcoded for test,  to just ensure that the underlying function is working as expected.